packages/now-cli/src/commands/alias/ls.js

```diff
@@ -84,13 +97,14 @@ export default async function ls(ctx, opts, args, output) {
       output.print(`${printPathAliasTable(rules)}\n`);
     }
   } else {
-    aliases.sort((a, b) => new Date(b.created) - new Date(a.created));
+    output.log(`aliases found under ${chalk.bold(contextName)} ${lsStamp()}`);
```

Comment:
```suggestion
    output.log(`plural('alias', aliases.length, true)} found under ${chalk.bold(contextName)} ${lsStamp()}`);
```

Is there a reason why you removed plural here?

Reply:
I removed it intentionally because if a user has let's say 500 aliases, and when he runs `now alias ls` we will return only 20, so we cannot just display `20 aliases found under <user>`, because that number does not represent the total number of aliases, but the number of aliases as part of the pagination.